PIM/API/02_GetStarted.md

```diff
@@ -0,0 +1,255 @@
+
+[comment]: <> (Check, ob das stimmt und ob hier nötig!)
+
+> [Caution]  
+    Before continuing, make sure you are not logged into your account. Otherwise, the authorization code will be linked to your user profile. That means that any API requests made subsequently will be made on your user's behalf.
+
+Following the example provided in [Step 1: Register the client app](#step-1-register-the-client-app), we describe the process in the https://oauthdebugger.com/ website. You can also use other third-party website or your own system. 
+
+*OAuth debugger*
+
+![OAuth debugger](../../Assets/Screenshots/PIM/API/OAuthDebugger.png "[OAuth debugger]")
+
+1. Enter the authorization URI (Uniform Resource Identifier) in the *Authorize URI (required)* field. In this case, your URI should look like this:
+
+    **https://your-workspace.actindo.com/Actindo.CoreModules.Auth.OAuth2.authorize**
+
+    Replace *your-workspace* with the name of your instance. Bear in mind that the authorization is account-specific.
+
+2. Enter **https://oauthdebugger.com/debug** in the *Redirect URI (required)* field.
+
+3. Enter your client ID in the *Client ID (required)* field. You have defined your client ID previously in the client app registration.
+
+4. Enter **profile** in the *Scope (required)* field.
+
+5. The *State* and *Nonce* fields are automatically generated.
+
+6. Click the *code* checkbox in the *Response type (required)* section.
+
+7. Select the *form_post* radio button in the *Response mode (required)* section. 
+
+8. Click the [SEND REQUEST] button.  
+    If the authentication flow is successful, OAuth debugger displays a success message with an authorization code, such as in the following screenshot: 
+
+    ![OAuth debugger success](../../Assets/Screenshots/PIM/API/OAuthDebuggerSuccess.png "[OAuth debugger success]")
+    
+9. Log in to the *Actindo Core1 Platform* with the user credentials (username and password) you want to authorize for API access.  
+    The authorization code is now linked to the logged in user.
+
+    ![Core1 Login](../../Assets/Screenshots/PIM/API/Core1Login.png "[Core1 Login]")
+
+[comment]: <> (Stimmt der letzter Schritt? Verstehe nicht ganz, wie es funktioniert.)
+
+[comment]: <> (Imogens Frage: Ist dann nicht Step 2: Create technical user for API access? Unsicher, ob hier noch einen Zwischenschritt fehlt. User ist schon erstellt, aber muss er bestimmte Rechte haben? Oder solange ein beliebiger Benutzer den authorization flow durchgeführt hat, kann er danach API requests senden?)
+
+
+### Step 3: Generate an access token
+
+Once you have obtained an authorization code, you can generate an access token for authentication.
+
+For the following steps, a cURL command line tool is used.
+
+The following parameters are required: 
+- Client ID, defined by you when registering the client app
+- Client secret, generated when saving the app registration, see [Step 1: Register the client app](#step-1-register-the-client-app)
```

Comment:
Vorschlag: 
-  Client secret. You have generated it when saving the app registration, see [Step 1: Register the client app](#step-1-register-the-client-app).